docs/api/plugins.md

```diff
@@ -439,6 +446,24 @@ uniform request distribution.  To enable this, you can pass in
 `options.tokensTable`, which is simply any Object that supports `put` and `get`
 with a `String` key, and an `Object` value.
 
+## Inflight Request Throttling
+
+```js
+const options = { limit: 600, server: server };
+options.resp = new require('restify-errors').InternalServerError();
```

Comment:
Since people copy paste snippets all the time, we should hoist the require up to the top and then call `new` on the constructor. Additionally, I thought this was a response? Why is it now an error object?